### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by CharelMarais**
Implemented routing

### CODE REVIEW

File: src/routes/Header.tsx [hunk 1]
```diff
@@ -18,15 +18,34 @@ export function Header() {
...
-          <button className="cursor-pointer w-10">
+          <div className="dropdown dropdown-bottom dropdown-end dropdown-hover">
+            <label tabIndex={0}>
+              <TbListSearch className="text-amber-500 text-3xl w-full h-full font-bold pr-2 -pl-2" />
+            </label>
+            <ul
+              tabIndex={0}
+              className="dropdown-content menu p-2 shadow bg-stone-800 text-amber-500 border-amber-500 border-2 border-b-0 border-t-0 rounded-box w-52"
+            >
+              <li>
```
Comment (MihlaliMgwebi) [inline src/routes/Header.tsx:29]: take a look at nesting your 'mangalist' routes.

File: src/routes/MangaPage.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { useLocation } from "react-router-dom";
+
+export function MangaPage() {
+  const location = useLocation();
+  const test = location.state;
+
+  console.log(test); // just here to test data for now
```
Comment (MihlaliMgwebi) [inline src/routes/MangaPage.tsx:7]: yessir. Thank you for the comment. Much appreciated!

**Review by MihlaliMgwebi — APPROVED** [review]
Looks really good Charel! Approved, with feedback.

File: src/routes/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+              </li>
+              <li>
+                <Link to="mangalist/followedCount">Most Popular </Link>
+              </li>
+              <li>
+                <Link to="mangalist/latestUploadedChapter">New Chapters</Link>
+              </li>
+            </ul>
+          </div>
+          {/* <button className="cursor-pointer w-10">
```
Comment (kphutiyagae) [inline src/routes/Header.tsx:43]: Would it be possible to remove commented out code if not necessary.

**Review by kphutiyagae — CHANGES_REQUESTED** [review]
Please address commented out code before merge.

File: src/routes/Header.tsx (hunk 3 above)
Comment (CharelMarais) [reply src/routes/Header.tsx:43]: I have made the adjustment

**Pull request merged**